WooCommerce/src/main/kotlin/com/woocommerce/android/config/FirebaseRemoteConfigRepository.kt

```
@@ -92,8 +93,14 @@ class FirebaseRemoteConfigRepository @Inject constructor(
                 emit(JetpackTimeoutPolicyVariant.valueOf(defaultValues[JETPACK_TIMEOUT_POLICY_VARIANT_KEY]!!))
             }
 
-    override fun getSimplifiedLoginVariant(): LoginVariant =
-        LoginVariant.valueOf(remoteConfig.getString(SIMPLIFIED_LOGIN_VARIANT_KEY).uppercase())
+    override fun getSimplifiedLoginVariant(): LoginVariant {
+        return try {
+            LoginVariant.valueOf(remoteConfig.getString(SIMPLIFIED_LOGIN_VARIANT_KEY).uppercase())
+        } catch (e: IllegalArgumentException) {
+            crashLogging.get().recordException(e)
```

This error shouldn't happen now with the default value, but we are using it just in case someone changes the configuration on the experiment without paying attention to the required values. In this case, it's better to report an error, but not block the user.